docs/ops/detection/ExperimentalDetectronDetectionOutput_6.md

```diff
@@ -0,0 +1,161 @@
+## ExperimentalDetectronDetectionOutput <a name="ExperimentalDetectronDetectionOutput"></a> {#openvino_docs_ops_detection_ExperimentalDetectronDetectionOutput_6}
+
+**Versioned name**: *ExperimentalDetectronDetectionOutput-6*
+
+**Category**: Object detection
+
+**Short description**: An operation *ExperimentalDetectronDetectionOutput* performs non-maximum suppression to generate 
+the detection output using information on location and score predictions.
+
+**Detailed description**: Operation doing next steps:
+
+1.  Applies deltas to boxes sizes;
+2.  If *class_agnostic_box_regression* is `true` then operation removes predictions for background classes;
+3.  Clips boxes to image;
+4.  Applies *score_threshold* on detection scores;
+5.  Applies non-maximum suppression class-wise with *nms_threshold*;
+6.  Operation returns *max_detections_per_image* detections if total number of detections is more than it, otherwise 
+returns total number of detections and the output tensor is filled with undefined values for rest output tensor 
+elements.
+
```

Comment:
Not clear how `post_nms_count` participates here. Please describe it.

Reply:
Updated.